src/azure-cli/azure/cli/command_modules/sql/_params.py

```diff
@@ -163,6 +163,18 @@ def get_location_type_with_default_from_resource_group(cli_ctx):
     options_list=['--managed-instance', '--mi'],
     help='Name of the Azure SQL managed instance.')
 
+kid_param_type = CLIArgumentType(
+    'kid',
```

Comment:
You need to remove this. 'dest' is not part of a CLIArgumentType.

Reply:
addressed